rpc/handlers.go

```diff
@@ -58,6 +58,12 @@ var (
 
 	// These errors can be only be returned by Juno-specific methods.
 	ErrSubscriptionNotFound = &jsonrpc.Error{Code: 100, Message: "Subscription not found"}
+
+	// TODO[pnowosie]: Update the error while specification describe it
+	ErrStorageProofNotSupported = &jsonrpc.Error{
+		Code:    42,
+		Message: "the node doesn't support storage proofs for blocks that are too far in the past. Use 'latest' as block id",
```

Comment:
nit: Juno only supports `starknet_getStorageProof` when block id is set to `latest`